### DISCUSSION TITLE
Translate to Russian

### DISCUSSION THREAD

**Discussion by Akrobs** [the-djmaze/snappymail#979] [General]
How can i help translate to Russian?
Every update im replace translation files...
Maybe im translate it for all?

**Comment by the-djmaze** [comment]
You can fork the project and then commit changes.
Then create a PR and it will be merged.

Else just upload your changes in a zip and i will

**Reply by webagroprom** [reply]
What is the name of the file where this form is located?

In which file can I make changes for translation ?

![1678919003573](https://user-images.githubusercontent.com/30408997/225456988-14c1a425-ae1a-4e36-b1be-bb69bfb5f47e.jpg)

**Reply by the-djmaze** [reply]
Currently none.
The location is plugins/nextcloud and more should become translatable there.
The extension was a fast wrapup for Nextcloud integration request.

**Reply by Akrobs** [reply]
> What is the name of the file where this form is located?
> 
> In which file can I make changes for translation ?
> 
> ![1678919003573](https://user-images.githubusercontent.com/30408997/225456988-14c1a425-ae1a-4e36-b1be-bb69bfb5f47e.jpg)

You can find it in file /you nc/data/appdata_snappymail\_data_\_default_\plugins\nextcloud\templates\PopupsNextcloudFiles.html

**Comment by webagroprom** [comment]
**Translated the en.ini file in ru.ini**

`/data/appdata_snappymail/_data_/_default_/plugins/nextcloud/langs
`

Created a branch, but could not upload changes there (writes no access to the repository, constantly asks for a password to add changes).

```
git branch rutranslate
git chackout rutranslate
git commit -m "RUtranslate create ru.ini"
git push origin rutranslate
```
Username for 'https://github.com': (ssh connection set up )

ZIP

[ru.ini.zip](https://github.com/the-djmaze/snappymail/files/10985434/ru.ini.zip)

ZIP files (ru): PopupsNextcloudFiles.html

**puth:**
`/you nc/data/appdata_snappymail_data__default_\plugins\nextcloud\templates\PopupsNextcloudFiles.html`

[PopupsNextcloudFiles.html.zip](https://github.com/the-djmaze/snappymail/files/10985432/PopupsNextcloudFiles.html.zip)
